gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -122,12 +123,9 @@ def _bigquery_to_vcf_shards(
   Also, it writes the meta info and data header with the call names to
   `vcf_header_file_path`.
   """
-  bq_source = bigquery.BigQuerySource(
-      query=_BASE_QUERY_TEMPLATE.format(
-          INPUT_TABLE='.'.join(bigquery_util.parse_table_reference(
-              known_args.input_table))),
-      validate=True,
-      use_standard_sql=True)
+  bq_source = bigquery.BigQuerySource(query=_form_customized_query(known_args),
```

Comment:
nit: Consider printing the query as logging.info (easier to debug). You'd need to store it, but I think that's ok and would be helpful as the queries get more complex.

Reply:
Done. This will be very helpful.